clojure-ts-mode.el

```diff
@@ -2032,6 +2032,22 @@ value is `clojure-ts-thread-all-but-last'."
                     "-"))))
     (user-error "No defun at point")))
 
+(defun clojure-ts-toggle-keyword-string ()
+  "Convert the string at point to a keyword, or vice versa."
+  (interactive)
+  (let ((node (treesit-thing-at-point 'sexp 'nested))
+        (pos (point)))
+    (cond
+     ((clojure-ts--string-node-p node)
+      (if (string-match-p " " (treesit-node-text node t))
+          (user-error "String should not contain spaces")
```

Comment:
Or something like "Cannot convert a string containing spaces to keyword" - seems more descriptive to me.